tests/javascript/README.md

```diff
@@ -0,0 +1,15 @@
+In order to run the tests you need to have mocha and chai installed. You can do it by running the following command
+Go into tests/javascript and run:
+```bash
+npm install mocha chai --save-dev
```

Comment:
Is there a way to install them from the package.json? (devDependencies)

Reply:
I see I added them to the package.json could you test if the install already works? @irlaec